src/infra/alarm/template.yaml

```diff
@@ -600,25 +600,6 @@ Resources:
 # Cloudwatch alarms for TxMA Latency
 #
 
-  AverageTxMALatencyOver1Sec:
```

Comment:
Is an average Latency of > 1 second something we should be aware of? Seems quite long although may be fine in context of AIS? @TaherKapasi

Reply:
Sounds good that's been reflected :)